jira_agile_metrics/config.py

```diff
@@ -390,7 +389,6 @@ def config_to_options(data, cwd=None, extended=False):
         for key in [
             'backlog_column',
             'committed_column',
-            'final_column',
```

Comment:
Will this not regress existing configurations that rely on this parameter?

Reply:
`final_column` was only used once in `ageingwip.py`, however, I changed the code to use `done_column` instead for the reasons described in https://github.com/DeloitteDigitalUK/jira-agile-metrics/pull/27#issuecomment-636308642. I'm confident to say that this won't break anything.